Automattic/jetpack
# AI Extension: show AI notices

~# require https://github.com/Automattic/jetpack/pull/32223~

This PR shows a Notice when AI needs to provide feedback to the user.
This is a tentative approach that uses the core Notice implementation.

Fixes #

## Proposed changes:

* AI Extension: show AI notices 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Create a Jetpack Form block instance
* Try to generate an error when requesting a suggestion
* Confirm the app shows the app Notice 

https://github.com/Automattic/jetpack/assets/77539/d20f68d7-0ba5-4465-9487-819d1c58b56d
